### DISCUSSION TITLE
REE patterns shifting to the right

### DISCUSSION THREAD

**Discussion by haranhe** [GeoPyTool/GeoPyTool#17] [Q&A]
Hello,
I have a problem which started only in the last version - the REE pattern shifts to the right and forming a blank tab left to the La - with values. I tested the samples file and they give the same result. All the other functions seems to work fine.

![REE](https://user-images.githubusercontent.com/77075305/111966199-527ab880-8aff-11eb-8f25-3a106429ad13.png)

thanks.

**Comment by cycleuser** [comment]
Thanks A LOT! GOT iT!

gonna repair right now

**Reply by ringosta** [reply]
i also have got problem with this. where i can find the fixed program?. thanks.

**Reply by cycleuser** [reply]
> i also have got problem with this. where i can find the fixed program?. thanks.

Update to latest version will fix this.

```Python
pip install cython numpy scipy matplotlib sympy pandas xlrd pyopengl BeautifulSoup4 pyqt5 scikit-learn requests tensorflow torch keras tqdm gym DRL -U
pip install geopytool -U
```


<img width="1183" alt="image" src="https://user-images.githubusercontent.com/6130092/137702010-5590cf29-e894-4d2b-a460-787afb65c247.png">

**Comment by cycleuser** [comment]
Fixed!

![2311616582985_ pic_hd](https://user-images.githubusercontent.com/6130092/112298230-c371ca00-8cd1-11eb-9f9c-eb592ba9b4b5.jpg)
